src/components/main/ItemsTable.tsx

```diff
@@ -89,6 +94,30 @@ const ItemsTable: FC<Props> = ({
   const { data: parentItem } = useItem(itemId);
   const { data: member } = useCurrentUserContext();
 
+  const { data: itemsTags } = useItemsTags(rows.map((r) => r.id).toJS());
+  const noStatusesToShow = !rows
+    .map((r) => {
+      const {
+        showChatbox = false,
+        isPinned = false,
+        isCollapsible = false,
+      } = r.settings || {
+        showChatbox: false,
+        isPinned: false,
+        isCollapsible: false,
+      };
+      return showChatbox || isPinned || isCollapsible;
+    })
+    .concat(
+      itemsTags?.map(
+        (itemTags) =>
+          isItemHidden({ tags: tagList, itemTags }) ||
+          isItemPublic({ tags: tagList, itemTags }) ||
+          isItemPublished({ tags: tagList, itemTags }),
+      ),
+    )
+    .some((e) => e === true);
```

Comment:
Does this mean that the computation is done twice, i.e. once for showing the statuses, and once here to compute the inverse? In which case, would it make sense to have a partition function that traverses the computations once and splits between the rows that do have statuses, and the ones that do not?